wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/kotlin/SourceSets.kt

```diff
@@ -23,7 +23,7 @@ import org.jetbrains.kotlin.gradle.plugin.KOTLIN_DSL_NAME
 import org.jetbrains.kotlin.gradle.plugin.KOTLIN_JS_DSL_NAME
 import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
 
-internal val AndroidSourceSet.kotlin: SourceDirectorySet?
+internal val AndroidSourceSet.kotlinSourceDirectorySet: SourceDirectorySet?
```

Comment:
Android itself added a property called `kotlin` so we have to pick another name

Reply:
Awww